camel-core/src/test/java/org/apache/camel/processor/MultiCastParallelAndStreamCachingWithEncryptionTest.java

```diff
@@ -0,0 +1,113 @@
+/*
```

Comment:
I think in Camel 2.x the license header may have 2 x stars in the beginning.

Can you run checkstyle

mvn compile -P sourcecheck

And verify

Reply:
I added the extra star